plugins/janus_streaming.c

```diff
@@ -1271,6 +1364,11 @@ struct janus_plugin_result *janus_streaming_handle_message(janus_plugin_session
 			if(error_code != 0)
 				goto plugin_response;
 		}
+
+		if(getifaddrs(&ifas) || ifas == NULL) {
+			JANUS_LOG(LOG_ERR, "Unable to acquire list of network devices/interfaces; some configurations may not work as expected...\n");
```

Comment:
Well it means the OS is not returning any network interfaces, i.e. claiming there *are* no network interfaces or we're not able to see them for some other reason. This is one of those things that one might file under "should not ever happen".